evaluator.measure/src/main/java/org/opencds/cqf/cql/evaluator/measure/common/MeasureEvaluator.java

```diff
@@ -437,8 +439,21 @@ protected void evaluateSdes(List<SdeDef> sdes) {
 
     // consider more complex expression in future
     private void inspectInstanceEvaluation(SdeDef sdeDef, ExpressionDef expressionDef) {
```

Comment:
I don't understand why we need to do inspection of the resources or of the ExpressionType. Why is `getEvaluatedResources` not enough to distinguish between constructed and retrieved resources?